### DISCUSSION TITLE
Request: power line / RFI artifacts (over-the-air)

### DISCUSSION THREAD

**Discussion by JasperT231** [ntsc-rs/ntsc-rs#204] [Ideas]
I use the luma / chroma noise functions to give an "over-the-air" look, but real OTA broadcasts also had more than just pure white noise. Sometimes the video had periodic noise like interference from powerlines or AC adapters or whatever.

It would show up as diagonal lines across the image, sometimes wavey. I could pre-bake this into a video, but it would be cool if I could do it in the program dynamically. 

https://ftp.unpad.ac.id/orari/library/library-sw-hw/amateur-radio/tvi/INTERFERENCE%20HANDBOOK.htm

After lots of searching, I found this old website with great examples of what I mean. (see pics: Ham/CB, FM Broadcast, Garage door opener)

**Comment by Asyahril395** [comment]
That's useful! So what do you think, @oisintheblue and @notdigidani ?

**Reply by notdigidani** [reply]
oh my god yes

**Reply by oisintheblue** [reply]
PEAK

**Comment by DanielWarner123** [comment]
Ghosting too.
